src/DescriptorsCommand.cc

```diff
@@ -269,6 +269,10 @@ int AddDescriptor::construct_protobuf(
         label = cmd["label"].asString();
         props[VDMS_DESC_LABEL_PROP] = label;
     }
+    else {
+        label = "None";
+        props[VDMS_DESC_LABEL_PROP] = label;
```

Comment:
Could you pull this line out of the if else?

Reply:
actually, I realized we have another mechanism to do that.